packages/runtime/container-runtime/src/opLifecycle/definitions.ts

```diff
@@ -19,7 +19,7 @@ export type BatchMessage = IBatchMessage & {
 /**
  * Batch interface used internally by the runtime.
  */
-export interface IBatch {
+export interface IBatch<TMessages extends BatchMessage[] = BatchMessage[]> {
```

Comment:
I'd make it ```<TMessages extends BatchMessage = BatchMessage>```, i.e. make templated argument be type of element, not of an array

Reply:
Responded elsewhere. The motivation was to express that Grouped Batching has a single message.